haddock-api/src/Haddock/Backends/Hoogle.hs

```diff
@@ -106,14 +108,14 @@ outWith p = f . unwords . map (dropWhile isSpace) . lines . p . ppr
         f [] = []
 
 out :: Outputable a => DynFlags -> a -> String
-out dflags = outWith $ showSDocUnqual dflags
+out dflags = outWith $ showSDoc dflags
 
 operator :: String -> String
 operator (x:xs) | not (isAlphaNum x) && x `notElem` "_' ([{" = '(' : x:xs ++ ")"
 operator x = x
 
 commaSeparate :: Outputable a => DynFlags -> [a] -> String
-commaSeparate dflags = showSDocUnqual dflags . interpp'SP
+commaSeparate dflags = showSDoc dflags . interpp'SP
```

Comment:
The name is in fact misleading currently. It's a leftover because before https://gitlab.haskell.org/ghc/ghc/-/commit/c8c18a106458c80ec0eb5108d11b4ed9e2bc7478 (2014) it was:

```haskell
showSDocUnqual :: DynFlags -> SDoc -> String
-- Only used by Haddock
showSDocUnqual dflags doc
 = renderWithStyle dflags doc (mkUserStyle neverQualify AllTheWay)
```

but since it's:

```haskell
showSDocUnqual :: DynFlags -> SDoc -> String
-- Only used by Haddock
showSDocUnqual dflags sdoc = showSDoc dflags sdoc
```

Reply:
Ahh, alright then. Fair enough.